src/coreclr/jit/simd.cpp

```diff
@@ -2192,7 +2182,7 @@ GenTree* Compiler::impSIMDIntrinsic(OPCODE                opcode,
             SpecialCodeKind op2CheckKind;
             if (simdIntrinsicID == SIMDIntrinsicInitArray || simdIntrinsicID == SIMDIntrinsicInitArrayX)
             {
-                op2CheckKind = SCK_RNGCHK_FAIL;
```

Comment:
I assume it is by design this is throwing a different exception now?

Reply:
Yes and approved by API review over e-mail.